design-system/package.json

```diff
@@ -9,12 +9,7 @@
     "directory": "design-system"
   },
   "homepage": "https://uikit.commercetools.com",
-  "keywords": [
-    "javascript",
-    "design system",
-    "react",
-    "uikit"
-  ],
+  "keywords": ["javascript", "design system", "react", "uikit"],
```

Comment:
this again? 😄

Reply:
It's changeset...I'll look into how they run the format, maybe we need to fix it there.

Reply:
I think this is the option that is causing the issue: https://github.com/atlassian/changesets/blob/f9ccfa3e9911718fcd95eff398ece64c8e9eef9b/packages/apply-release-plan/src/index.ts#L116

<img width="371" alt="image" src="https://user-images.githubusercontent.com/1110551/98129022-ee8d8580-1eb8-11eb-8a18-f263510257b5.png">